pyproject.toml

```diff
@@ -14,7 +14,6 @@ members = [
     "githubscript",
     "iscript",
     "landoscript",
-    "notarization_poller",
```

Comment:
Oops, is that something I forgot when I removed it? Surprised nothing was complaining about it